### PR TITLE
Tagging of individual certificates/templates

### LINKED ISSUE #51: Labelling/Tagging of individual certificates/templates
Certificates/Templates should have the ability to be labelled/tagged. Each certificate/template may have none, one, or multiple tags/labels.

### PR DESCRIPTION
**Pull request by MitanshiKshatriya**
Closes #51
Added tags field to Template Schema and Certificate Schema  
1. Certificate Schema
```
title: {
		type: String,
		default: 'Certificate'
	},
	tags:[String], // Added tags field
	template: {
		type: String,
		required: true
	},
````
2. Template Schema
```
....
tags:[String],
...
```

### CODE REVIEW

File: src/models/certificate.js [hunk 1]
```diff
@@ -17,6 +17,7 @@ const certificateSchema = new Schema({
 		type: String,
 		default: 'Certificate'
 	},
+	tags:[String],
```
Comment (paramsiddharth) [inline src/models/certificate.js:20]:
```suggestion
	tags: [String],
```

File: src/models/template.js [hunk 2]
```diff
@@ -17,6 +17,7 @@ const templateSchema = new Schema({
 		type: String,
 		required: true
 	},
+	tags:[String],
```
Comment (paramsiddharth) [inline src/models/template.js:20]:
```suggestion
	tags: [String],
```

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
Please make the suggested changes for a more consistent code style.

**Comment by MitanshiKshatriya** [comment]
Ok got it

**Comment by MitanshiKshatriya** [comment]
I have made the changes please check.

**Review by paramsiddharth — APPROVED** [review]
Looks good to me! We'll proceed with further updates pertaining to the usage of tags.

**Pull request merged**